elide-datastore/elide-datastore-hibernate/src/main/java/com/yahoo/elide/core/filter/HQLFilterOperation.java

```diff
@@ -1,198 +0,0 @@
-/*
- * Copyright 2016, Yahoo Inc.
- * Licensed under the Apache License, Version 2.0
- * See LICENSE file in project root for terms.
- */
-package com.yahoo.elide.core.filter;
-
-import com.yahoo.elide.core.exceptions.InvalidPredicateException;
-import com.yahoo.elide.core.exceptions.InvalidValueException;
-import com.yahoo.elide.core.filter.FilterPredicate.FilterParameter;
-import com.yahoo.elide.core.filter.expression.AndFilterExpression;
-import com.yahoo.elide.core.filter.expression.FilterExpression;
-import com.yahoo.elide.core.filter.expression.FilterExpressionVisitor;
-import com.yahoo.elide.core.filter.expression.NotFilterExpression;
-import com.yahoo.elide.core.filter.expression.OrFilterExpression;
-
-import com.google.common.base.Preconditions;
-import com.google.common.base.Strings;
-
-import java.util.List;
-import java.util.function.Function;
-import java.util.stream.Collectors;
-
-/**
- * FilterOperation that creates Hibernate query language fragments.
- */
-public class HQLFilterOperation implements FilterOperation<String> {
```

Comment:
I see this branch is merged into master directly. Maybe we should deprecate it first and remove it in 5.0, so that we give downstream projects some buffer time